### PR TITLE
NEW: Added command descriptions to q2-fmt command 

### LINKED ISSUE #63: Add parameter, method and output descriptions

### PR DESCRIPTION
**Pull request by cherman2**

### CODE REVIEW

**Review by hagenjp — COMMENTED** [review]
These changes look good to me @cherman2!

**Comment by cherman2** [comment]
closes #63 

no rush on this @hagenjp ( I know you have the report, classifiers etc. ), just linking my existing issue!

File: q2_fmt/plugin_setup.py [hunk 1]
```diff
@@ -82,6 +83,7 @@
                    ' The "all-pairwise" comparison defines Group A as all'
                    ' groups in `reference_column` and `control_column`, and'
                    ' Group B is all timepoints in `time_column`.',
+        'distance_to': 'what reference type to calculate distance to against',
```
Comment (hagenjp) [inline q2_fmt/plugin_setup.py]:
```suggestion
        'distance_to': 'What reference type to calculate distance to against',
```

File: q2_fmt/plugin_setup.py [hunk 2]
```diff
@@ -143,10 +148,15 @@
...
-    outputs=[('timepoint_dists', Dist1D[T_nested, T_dependence]),
+    outputs=[('timepoint_dists', Dist1D[Ordered, T_dependence]),
              ('reference_dists', Dist1D[Unordered, Independent])],
+    input_descriptions={
+        'diversity_measure': 'diversity metric to put in a long form data'
+                             ' structure'
+    },
     parameter_descriptions={
         'metadata': 'The sample metadata.',
+        'distance_to': 'what reference type to calculate distance to against',
```
Comment (hagenjp) [inline q2_fmt/plugin_setup.py]:
```suggestion
        'distance_to': 'What reference type to calculate distance to against',
```

File: q2_fmt/plugin_setup.py [hunk 3]
```diff
@@ -143,10 +148,15 @@
                 'group_column': T_group, 'control_column': Str,
                 'filter_missing_references': Bool, 'where': Str,
                 'baseline_timepoint': Str, 'where': Str},
-    outputs=[('timepoint_dists', Dist1D[T_nested, T_dependence]),
+    outputs=[('timepoint_dists', Dist1D[Ordered, T_dependence]),
```
Comment (hagenjp) [inline q2_fmt/plugin_setup.py]:
```suggestion
    outputs=[('timepoint_dists', Dist1D[T_nested, T_dependence]),
```

File: q2_fmt/plugin_setup.py [hunk 4]
```diff
@@ -206,19 +220,49 @@
...
+        'subject_column': 'The column within the `metadata` that contains the'
+                          ' subject ID to be tracked against timepoints.',
+        'filter_missing_references': 'Filter out references contained within'
+                                     ' the metadata that are not present'
+                                     ' in the table.'
+                                     ' Default behavior is to raise an error.',
+        'drop_incomplete_subjects': 'Filter out subjects that do not have'
+                                    ' a sample at every timepoint.'
+                                    ' Default behavior is to raise an error.',
+        'drop_incomplete_timepoint': 'Filter out a list of provided timepoint.'
```
Comment (hagenjp) [inline q2_fmt/plugin_setup.py]:
```suggestion
        'drop_incomplete_timepoint': 'Filter out provided timepoint.'
```

File: q2_fmt/plugin_setup.py [hunk 5]
```diff
@@ -206,19 +220,49 @@
...
+                                     ' This will be preformed before'
```
Comment (hagenjp) [inline q2_fmt/plugin_setup.py]:
```suggestion
                                     ' This will be performed before'
```

**Pull request merged**